ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/vm/MessageFrame.java

```diff
@@ -29,10 +29,7 @@
 import tech.pegasys.pantheon.util.uint.UInt256;
 import tech.pegasys.pantheon.util.uint.UInt256Value;
 
-import java.util.Deque;
-import java.util.EnumSet;
-import java.util.HashSet;
-import java.util.Set;
+import java.util.*;
```

Comment:
This is triggering ErrorProne: https://jenkins.pegasys.tech/job/Pantheon/job/PR-1258/5/execution/node/38/log/